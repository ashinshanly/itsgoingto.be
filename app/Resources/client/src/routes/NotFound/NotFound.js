import React from 'react'
import Back from 'components/Back/Back'
import './NotFound.scss'

export const NotFound = () => (
  <div>
    <Back />
    <div className='container header-container answer-header-container'>
      <div className='header center-text'>
        <h1 className='not-found'>
          Oh no!<br />
          <span>It looks like we couldn't find that poll.</span>
        </h1>
      </div>
    </div>
  </div>
)

export default NotFound
