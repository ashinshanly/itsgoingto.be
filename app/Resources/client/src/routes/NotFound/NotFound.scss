h1.not-found {
  font-size: 5em;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  position: fixed;

  span {
    font-size: 0.6em;
  }
}
