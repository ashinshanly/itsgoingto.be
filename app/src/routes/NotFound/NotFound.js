import React from 'react'
import Back from 'components/Back'
import './NotFound.scss'

export function NotFound () {
  return (
    <div>
      <Back />
      <div className='container header-container answer-header-container'>
        <div className='header center-text'>
          <h1 className='not-found'>
            Oh no!<br />
            <span>{ 'It looks like we couldn\'t find that poll.' }</span>
          </h1>
        </div>
      </div>
    </div>
  )
}

export default NotFound
