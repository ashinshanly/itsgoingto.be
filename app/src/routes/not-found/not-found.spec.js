/* global expect */
import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './not-found'

describe('(Route) NotFound', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<NotFound />)
  })

  describe('(Render)', () => {
    it('matches snapshot', () => {
      expect(wrapper).toMatchSnapshot()
    })
  })
})