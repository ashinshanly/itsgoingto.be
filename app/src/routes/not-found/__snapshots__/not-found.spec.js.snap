// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Route) NotFound (Render) matches snapshot 1`] = `
<div>
  <Back />
  <div
    className="container header-container answer-header-container"
  >
    <div
      className="header center-text"
    >
      <h1
        className="not-found"
      >
        Oh no!
        <br />
        <span>
          It looks like we couldn't find that poll.
        </span>
      </h1>
    </div>
  </div>
</div>
`;
